Sources/SwiftToolchain/Toolchain.swift

```diff
@@ -226,9 +221,9 @@ public final class Toolchain {
 
     terminal.write("\nBuilding the project before spinning up a server...\n", inColor: .yellow)
 
-    let builderArguments = try [
+    let builderArguments = [
       swiftPath.pathString, "build", "-c", isRelease ? "release" : "debug", "--product", product,
-      "--enable-test-discovery", "--destination", destination ?? inferDestinationPath().pathString,
+      "--enable-test-discovery", "--triple", "wasm32-unknown-wasi", "-Xswiftc", "-static-stdlib",
```

Comment:
QQ: is `-static-stdlib` now required in the latest snapshots? Should we patch SwiftPM so that it passes it to the driver automatically without requiring stand-alone SwiftPM users and `carton` to do it?

Reply:
Good question. I'm thinking about whether we should make static-stdlib to be the default for wasm target.

Reply:
I think we should, it essentially was enabled by default all this time, at least in the outcome.😄 And can you even build anything useful without linking stdlib statically?

Reply:
ok, I'll tackle this as my next task. Anyway, can we merge this pr without making the option default at now ?

Reply:
Absolutely, thanks again for all this work!